baseapp-profiles/baseapp_profiles/graphql/mutations.py

```diff
@@ -210,3 +246,4 @@ class ProfilesMutations(object):
     profile_create = ProfileCreate.Field()
     profile_update = ProfileUpdate.Field()
     profile_delete = ProfileDelete.Field()
+    role_update = RoleUpdate.Field()
```

Comment:
change to `profile_role_update` to make it more organized and explicit about what this is about in the graphql mutations